demo/src/routes/menu/CollapsibleSection.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+	import {collapseVerticalTransition} from '@agnos-ui/core/services/transitions/bootstrap';
+	import {createTransition} from '@agnos-ui/core/services/transitions/baseTransitions';
+	import {writable} from '@amadeus-it-group/tansu';
+
+	export let headerText;
+	const paramTransition$ = writable(collapseVerticalTransition);
+	const paramAnimation$ = writable(true);
+	const paramVisible$ = writable(false);
+
+	const {
+		stores: {visible$},
+		api: {toggle},
+		directives: {directive},
+	} = createTransition({
+		props: {
+			animation: paramAnimation$,
+			visible: paramVisible$,
+		},
+	});
+</script>
+
+<div class="mb-2">
+	<button class="btn btn-light btn-lg p-1 d-flex align-items-center btn-header" aria-expanded={$visible$} on:click={() => toggle()}
+		>{headerText}
+		<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 100 100">
+			<rect class="horizontal" x="20" y="40" width="60" height="20" fill="black" />
+			<rect class="vertical" class:expanded={$visible$} x="40" y="20" width="20" height="60" fill="black" />
+		</svg>
+	</button>
+	<div class="contents" use:directive={{transition: $paramTransition$, animation: $paramAnimation$}}>
```

Comment:
wanted to remove the content with an if-block when it is hidden, however, then the sveltekit prerendering is not working automatically, so maybe not worth for a list of links WDYT?

Reply:
I do agree with you, moreover I suggested to open menus by default, which it means you load all the menu at page load.